runners/mlcube_docker/mlcube_docker/tests/test_docker_runner.py

```diff
@@ -51,11 +52,11 @@ def tearDown(self) -> None:
         Shell.sync_workspace = self.sync_workspace
 
     @unittest.skipUnless(_HAVE_DOCKER, reason="No docker available.")
+    @patch("io.open", mock_open(read_data=_MLCUBE_DEFAULT_ENTRY_POINT))
```

Comment:
It's good to keep this pathing feature inside the function (explanation - [772764](https://github.com/mlcommons/mlcube/commit/0d9c4bea562e4816ae027e1c6835339f69d7de1f)):
```python
with patch("io.open", mock_open(read_data=_MLCUBE_DEFAULT_ENTRY_POINT)):
    mlcube: DictConfig = MLCubeConfig.create_mlcube_config(
        "/some/path/to/mlcube.yaml", runner_config=Config.DEFAULT, runner_cls=DockerRun
    )
```